samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx

```diff
@@ -85,7 +85,7 @@ class Dashboard extends Component {
 
     // Exchange client token with server token and fetch the pinned message details.
     exchangeClientTokenForServerToken = async (token) => {
-        var id = "12345";
+        var id = "ChatId";  //Replace {{ChatId ex - 19:75e4d14a2c2c4644a7adaf4e3976d17b@thread.v2}}  
```

Comment:
chat id needs to be static or dynamic value ?

Reply:
Replace the placeholder 'chatid' with the actual chat ID.

Reply:
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/51744568-8c16-4056-ac52-a7ca550f5688)